tests/functional/disaster-recovery/regional-dr/test_cnv_failover_and_relocate_discovered_apps.py

```diff
@@ -0,0 +1,186 @@
+import logging
+from time import sleep
+
+from ocs_ci.deployment.cnv import CNVInstaller
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import acceptance, tier1
+from ocs_ci.framework.pytest_customization.marks import turquoise_squad, rdr
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.helpers.cnv_helpers import run_dd_io
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity_vm
+
+logger = logging.getLogger(__name__)
+
+
+@rdr
+@acceptance
+@tier1
+@turquoise_squad
+class TestCNVFailoverAndRelocateWithDiscoveredApps:
+    """
+    Test CNV Failover and Relocate with Discovered Apps
+
+    """
+
+    def test_cnv_failover_and_relocate_discovered_apps(
+        self, discovered_apps_dr_workload_cnv
+    ):
+        """
+        Tests to verify cnv application failover and Relocate with Discovered Apps
+        There are two test cases:
+            1) Failover to secondary cluster when primary cluster is UP
```

Comment:
With failover, we should test node down scenario for happy path

Reply:
this will be covered in next test